cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -476,3 +479,25 @@ def _get_creation_failures(cluster_status, cfn_stack):
         return None
     failure_code, failure_reason = cfn_stack.get_cluster_creation_failure()
     return [Failure(failure_code=failure_code, failure_reason=failure_reason)]
+
+
+def _get_details(cfn_stack, verbose):
```

Comment:
nit: this can be one line: return _get_alarms_details(cfn_stack) if verbose else None

Reply:
Revised. Good point!